models.py

```diff
@@ -1253,6 +1254,14 @@ class CognitionMessage(Base):
 
     scope_dict_diff_previous_conversation = Column(JSON)
     scope_dict_diff_new = Column(JSON)
+    version_id = Column(
+        # pipeline version with which the version was created, can be null if the version isn't available anymore (e.g. deleted)
```

Comment:
version= message in the second part here?

Reply:
no the actual pipeline version is meant so if it's deleted this will cascade to null. 

Maybe I'm misunderstanding the question 🤔

Reply:
pipeline version with which the **version** was created = pipeline version with which the **message** was created.

Or what is the version, the  version of the message was created with?

Reply:
got it :D changed